packages/wethegit-components-cli/src/registry-index.ts

```diff
@@ -33,6 +33,13 @@ const BREAKPOINTS_TYPE: Registry = {
   dontShowOnPrompt: true,
 }
 
+/* HOOKS */
+const USE_BREAKPOINTS: Registry = {
+  name: "use-breakpoints",
+  category: "hook",
+  dontShowOnPrompt: true,
```

Comment:
@andrewrubin why not?

Reply:
Since breakpoints and all their opinions come along with the styles by default, I figured this should too. Happy either way

Reply:
Let's add it, cuz otherwise we'll need to do some tweaks to the CLI and I think for now it's okay.

Reply:
Let's make sure that the `BreakpointSnipe` depends on it too

Reply:
done and done. Good catch